stream-legal-entity/legal-entity-core/src/main/java/com/backbase/stream/LegalEntitySaga.java

```diff
@@ -902,8 +902,7 @@ private Mono<LegalEntityTask> processSubsidiaries(LegalEntityTask streamTask) {
 
         Flux<LegalEntity> subsidiaries = parentLegalEntity.getSubsidiaries() == null
             ? Flux.empty()
-            : Flux.fromIterable(parentLegalEntity.getSubsidiaries());
-
+            : Flux.fromStream(parentLegalEntity.getSubsidiaries().stream().filter(LegalEntity::getCreateInDBS));
```

Comment:
just to confirm, do you only expect this to work on subsidiaries, never on the root legal entity of that specific payload? if so, shouldn't we make it clearer in the description and in the flags?

Reply:
okay yes it is basically for subsidaries and if root itself filtered which means nothing to do in such case no need to invoke this at all. this can be mentioned in description.